include/psen_scan_v2/tenth_of_degree.h

```diff
@@ -28,12 +28,17 @@ class TenthOfDegree
     return TenthOfDegree(radToTenthDegree(angle_in_rad));
   }
 
+  static uint16_t toPositiveValue(const TenthOfDegree& tenth_of_degree)
+  {
+    return tenthDegreeToPositiveTenthDegree(tenth_of_degree.value());
+  }
+
 public:
-  explicit constexpr TenthOfDegree(const uint16_t& tenth_of_degree) : tenth_of_degree_(tenth_of_degree)
+  explicit constexpr TenthOfDegree(const int32_t& tenth_of_degree) : tenth_of_degree_(tenth_of_degree)
   {
   }
 
-  constexpr uint16_t value() const
+  constexpr int32_t value() const
```

Comment:
I feel like `value()` and `toPositiveValue` should return the same type

Reply:
I renamed `toPositiveValue` to `toUnsigned`. I hope now the intention is clear.